pkg/flow/states/parallel.go

```diff
@@ -329,7 +329,7 @@ func (logic *parallelLogic) processActionResults(ctx context.Context, children [
 
 		} else if results.ErrorMessage != "" {
 
-			logic.Log(ctx, "Branch %d crashed due to an internal error: %s", idx, results.ErrorMessage)
+			logic.SendToLogger(ctx, "error", "Branch %d crashed due to an internal error: %s", idx, results.ErrorMessage)
```

Comment:
All calls to `SendToLogger` are using strings to set log level. These should be constants.